tensorpipe/test/channel/cuda_ipc/kernel.cu

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <cuda.h>
+
+__global__ void _slowKernel(char* ptr, int sz) {
```

Comment:
Add a comment here on what it does and why we need it? Or perhaps even in the header file.